api/can_api_v2_definition.py

```diff
@@ -121,6 +133,45 @@ class ActualsTimeseriesRow(Actuals):
     date: datetime.date = pydantic.Field(..., description="Date of timeseries data point")
 
 
+class MetricAnomaly(base_model.APIBaseModel):
```

Comment:
Since we have a concept of metrics elsewhere do you think `DataAnomaly` or maybe `AnomalyAnnotation` would be more clear?

Reply:
ah I was thinking of metrics as the `metrics` and `metricsTimeseries` fields

Reply:
I dunno. Changed to `AnomalyAnnotation`.